server/lib/src/be/idl_sqlite.rs

```diff
@@ -205,12 +205,15 @@ pub(crate) trait IdlSqliteTransaction {
         let mut stmt = self
             .get_conn()?
             .prepare(&format!(
```

Comment:
This really should be 

```rust
.prepare(r#"SELECT DISTINCT NAME from sqlite_master where type="table" AND name = :tname"#)

Reply:
rowid is a faster lookup, but you're correct to select on type=table. I've made that change.